object-arrays/js-objects-and-arrays_objects-and-arrays/index.js

```diff
@@ -50,27 +50,27 @@ const people = [
 // EXERCISE 1
 // Change the value of the variable nameOfFirstPerson to equal the name of the first person.
 
-const nameOfFirstPerson = "change me";
+const nameOfFirstPerson = "Homer"; 
```

Comment:
I think the purpose of the exercise is not to hardcode values but use dot or square bracket notation to access property values from an object.